kuma/wiki/tests/test_templates.py

```diff
@@ -245,7 +246,7 @@ def test_revision_view(self):
         """Load the revision view page and verify the title and content."""
         d = _create_document()
         r = d.current_revision
-        r.created = datetime(2011, 1, 1)
+        r.created = datetime(2011, 1, 1, tzinfo=timezone.utc)
```

Comment:
You should as @jezdez it looks like he made the change in https://github.com/mozilla/kuma/commit/c276efff168af976de499c3b1a4b46717417bff4 ;)